app/views/presentations/_sign_up_link.html.erb

```diff
@@ -0,0 +1,5 @@
+<%=
+  if event&.upcoming?
+    link_to 'Sign up', new_presentation_path(prepopulated_event: event.id)
```

Comment:
Instead of `prepopulated_event` could we just use `event`? Or even `event_id` as that's what we're passing.